From c77a7fc75b7f8fd6b482e773874012add962624e Mon Sep 17 00:00:00 2001
From: Antonio Borneo <antonio.borneo@foss.st.com>
Date: Tue, 4 Jul 2023 15:29:45 +0200
Subject: [PATCH 0373/1141] arm64: dts: st: fix button-user on board
 stm32mp257f-ev1

Only button named USER1 is available for Linux; USER2 is allocated
to Cortex-M33.

Change the node button-user-2 to button-user-1.

Signed-off-by: Antonio Borneo <antonio.borneo@foss.st.com>
Change-Id: Ib67bb5230847db471ba25dc826221c6c0b1c4963
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/316280
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-ev1.dts | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
@@ -48,10 +48,10 @@
 	gpio-keys {
 		compatible = "gpio-keys";
 
-		button-user-2 {
-			label = "User-2";
-			linux,code = <BTN_2>;
-			gpios = <&gpiog 8 GPIO_ACTIVE_HIGH>;
+		button-user-1 {
+			label = "User-1";
+			linux,code = <BTN_1>;
+			gpios = <&gpiod 2 GPIO_ACTIVE_HIGH>;
 			status = "okay";
 		};
 	};
